marklogic-data-hub-central/ui-custom/src/components/ImageGallery/ImageGallery.test.tsx

```diff
@@ -0,0 +1,84 @@
+
+import {render, waitFor} from "@testing-library/react";
+import {DetailContext} from "../../store/DetailContext";
+import ImageGallery from "./ImageGallery";
+
+const configMultiple = {
+  component: "ImageGallery",
+  config: {
+    style: {
+      height: "150px",
+      width: "150px"
+    },
+    images: {
+      url: {
+        arrayPath: "person.images.image",
+        path: "url"
+      }
+    },
+    metadata: {},
+    download: true
+  }
+};
+
+const detail = {
+  person: {
+    images: {
+      "image": [
+        {
+          "url": "https://cdn1.marklogic.com/wp-content/uploads/2018/01/Diane-Burley-MarkLogic-2015.jpg",
```

Comment:
Can we simplify the test data? Have an array of 2 image objects (instead of 4). Also use dummy URLs, http://example.org/image1.jpg, http://example.org/image2.png